vaccine-standby/assets/index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <link rel="stylesheet" href="/styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>COVID-19 Vaccine Standby List</h1>
+      <img class="standby-list-diagram" src="/standby-list-diagram.png" />
+      <p>This Quick Deploy app is designed for public health agencies to create a COVID-19 vaccine eligibility standby list for their city or state.
+      Residents send an SMS to a pre-configured phone number, and are asked a series of demographic questions informed by the CDC's vaccine rollout guidelines powered by a Twilio Studio Flow.
+       
+      <p>This data is meant to help ensure vaccinations are administered as quickly as possible as supply becomes available. With this template, your agency can create a standardized list of eligible
+      vaccine recipients. In exchange for sharing this information, residents are notified when it is their turn to receive the vaccine.</p>
+    </header>
+    <div role="main">
+      <section>
+        <div id="deploy-flow" style="display: none;" class="deploy-studio-flow">
+          <h3>Finish configuring your app</h3>
+          <p>Your app is deployed, but you need to finish configuring it to get it working. Click the button below to automatically run the following deployment steps:</p>
+          <ul>
+            <li>Deploy Vaccine Standby List Studio Flow to power SMS chatbot</li>
```

Comment:
maybe reword this to something like `Deploy a Twilio Studio Flow to power your Vaccine Standby List SMS chatbot`

Reply:
changed 👍